tests/jballerina-unit-test/src/test/java/org/ballerinalang/test/expressions/builtinoperations/FreezeAndIsFrozenTest.java

```diff
@@ -317,6 +317,13 @@ public void testFrozenTupleUpdate() {
         BRunUtil.invoke(result, "testFrozenTupleUpdate", new BValue[0]);
     }
 
+    @Test(expectedExceptions = BLangRuntimeException.class,
+            expectedExceptionsMessageRegExp = "error: \\{ballerina/lang.array}InvalidUpdate \\{\"message\":" +
+                    "\"modification not allowed on readonly value\".*")
```

Comment:
Can't/shouldn't we get rid of this? Can you verify if the test is running? I was under the impression this should fail because we are trapping the panic in testFrozenRecursiveTupleUpdate.